target/linux/ramips/image/mt7620.mk

```diff
@@ -804,6 +804,23 @@ define Device/netgear_wn3000rp-v3
 endef
 TARGET_DEVICES += netgear_wn3000rp-v3
 
+define Device/netgear_wn3100rp-v2
+  SOC := mt7620a
+  IMAGE_SIZE := 7872k
+  NETGEAR_HW_ID := 29764883+8+0+32+2x2+0
+  NETGEAR_BOARD_ID := WN3100RPv2
+  BLOCKSIZE := 4k
+  IMAGES += factory.bin
+  KERNEL := $(KERNEL_DTB) | uImage lzma | pad-offset 64k 64 | \
+	append-uImage-fakehdr filesystem
+  IMAGE/factory.bin := $$(sysupgrade_bin) | check-size | netgear-dni
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_MODEL := WN3100RP
+  DEVICE_VARIANT := v2
+  SUPPORTED_DEVICES += wn3100rpv2
```

Comment:
Please drop this line entirely unless it affects the factory image installation from the stock firmware.

Reply:
Sorry, but I'm confused about which line youre are talking about ? The whole block or a specific one VENDOR/MODEL/VARIANT or SUPPORTED_DEVICES ?

Reply:
I mean only `SUPPORTED_DEVICES`.

Reply:
Ok, Ive applied changes (including dtsi renaming) on my local branch. I'll test recompilation, Web UI Flash and sysupgrade and I'll push an update once I've validated the new images.